### PR TITLE
add simple-ui-components-in-svelte

### PR DESCRIPTION
**Pull request by tborychowski**

### CODE REVIEW

**Comment by MacFJA** [comment]
There were an issue with the preview deployment, which have been solved.
Can you rebase your branch ?

---

Also the `lint` task failed.
You can run `npm run format` to fix it

**Comment by tborychowski** [comment]
Updated my fork with the latest, thanks.
The `npm run format` doesn't change anything.

File: src/routes/components/components.json [hunk 1]
```diff
@@ -2827,5 +2827,15 @@
...
 		"stars": 0
+	},
+	{
+		"title": "simple-ui-components-in-svelte",
+		"url": "https://perfect-things.github.io/simple-ui-components-in-svelte/",
+		"description": "A set of simple & flexible UI components",
+		"npm": "simple-ui-components-in-svelte",
+		"addedOn": "2023-01-24",
+		"category": "Forms & User Input",
+		"tags": ["forms", "UI"],
```
Comment (MacFJA) [inline src/routes/components/components.json]: The tag UI doesn't exist

Comment (tborychowski) [reply src/routes/components/components.json]: That's a pity. It's a very useful tag 😉 
Removed.

Comment (MacFJA) [reply src/routes/components/components.json]: New tag can be added, but they need to be thoroughly thinked.
If a new tag is added, all other components need to be review to check if they can have it too.
And if almost all of them have it, the tag don't have a lot of meaning for the end user that want to filter (i.e. if adding the filter remove 2 items in a list of 300, it's worth nothing). And in the contrary, if a tag match almost nothing, it's either a very good filter or a terrible one.

**Pull request merged**